observer.cpp

```diff
@@ -10,21 +10,29 @@ observer::observer()
 bool operator==(const all_params& lhs, const all_params& rhs)
 {
     return lhs.i_p.net_par.net_arc ==  rhs.i_p.net_par.net_arc &&
-           lhs.p_p.mut_rate == rhs.p_p.mut_rate &&
```

Comment:
We need to fix the setting of your Qt (or my) so that indentation is the same, otherwise we keep getting these useless changes that confuse the pull requests